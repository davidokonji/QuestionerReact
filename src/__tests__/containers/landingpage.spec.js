import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../../containers/LandingPage.container';

describe('Landing Page', () => {
  test('should render landing page correctly', () => {
    const wrapper = shallow(<LandingPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
