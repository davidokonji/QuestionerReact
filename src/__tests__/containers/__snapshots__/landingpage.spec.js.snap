// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page should render landing page correctly 1`] = `
<div>
  <Header
    navClass="nav_header"
    page="landingpage"
  />
  <div
    className="landingContainer"
  >
    <div
      className="topLanding"
    >
      <div
        className="top"
      >
        <img
          alt="landingimage"
          className="top_img"
          src="../Assets/svg/undraw_teaching_f1cm.svg"
        />
      </div>
      <Steps />
    </div>
    <div
      className="bottomLanding"
    >
      <div
        className="detail"
      >
        <div
          className="detail_img"
        >
          <img
            alt=""
            src="../Assets/svg/undraw_mind_map_cwng.svg"
          />
        </div>
        <div
          className="detail_text"
        >
          <p
            className="detail_title"
          >
            Questioner
          </p>
          <p
            className="detail_sub_text"
          >
            Live your best life. Your One stop for meetup events
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
